src/app/sidebar/container.tsx

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react'
+import { NavLink } from 'react-router-dom'
 
 export default class Sidebar extends React.Component<any, any> {
+
   render() {
     return (
-      <div className="c-sidebar">
-          <div className="c-sidebar__nav">
-              <nav className="c-nav">
-                  <div className="c-nav__title">Navigation</div>
-                  <ul className="c-nav__items">
-                      <li>Overview</li>
-                      <li>Payments</li>
-                      <li className="active">Transactions</li>
-                      <li>Address book</li>
+      <div className='c-sidebar'>
+          <div className='c-sidebar__nav'>
+              <nav className='c-nav'>
+                  <div className='c-nav__title'>Navigation</div>
+                  <ul className='c-nav__items'>
+                    <li><NavLink exact to='/'> Overview</NavLink></li>
```

Comment:
do we highlight the nav when the location is corresponding to the nav path? if yes then we should have add activeClass

Reply:
yes the nav link should be active to indicate the current page, the active class will work after the next deploy of the css (this weekend)

Reply:
check the backlog of https://github.com/Electra-project/Electra-Styleguide/projects/1 to see whats planned for styles

Reply:
this is now fixed.